libcontainer/state_linux.go

```diff
@@ -65,6 +70,34 @@ func destroy(c *Container) error {
 	return err
 }
 
+func restoreNetworkDevices(c *Container) error {
+	if !c.config.Namespaces.Contains(configs.NEWNET) {
+		return nil
+	}
+	nsPath := c.config.Namespaces.PathOf(configs.NEWNET)
+	// if there is no network namespace configured
+	// try to get the init process pid to obtain it.
+	if nsPath == "" {
+		if c.initProcess.pid() > 0 {
+			nsPath = fmt.Sprintf("/proc/%d/ns/net", c.initProcess.pid())
+		} else {
+			return nil
+		}
```

Comment:
@rata I need help here, I can not find the best place to restore the network devices when the namespace is created by runc, it seems it has to be at one point where the init process is still running

Reply:
@aojea what do you mean "the init process is still running"? Which process?

state_linux.go is probably to know the state of running containers, you should check to standard_init_linux.go, for example